### ISSUE TITLE
Read parameters from file

### ISSUE THREAD

**Issue by yissachar** [InQuicker/ktmpl#7]
Instead of doing:

```
ktmpl example.yml --parameter MONGODB_USER carl --parameter MONGODB_PASSWORD secret
```

I want to be able to have a parameters file:

```
MONGODB_USER: carl
MONGODB_PASSWORD: secret
```

And pass that to `ktmpl`:

```
ktmpl example.yml -f params.yml
```

**Comment by jimmycuadra** [comment]
Good idea!

**Comment by solher** [comment]
I would definitely love it too. We would be really close to the equivalent of a Docker Compose `.env` file for Kubernetes!

**Comment by seh** [comment]
See #14.

**Comment by jimmycuadra** [comment]
This was released in 0.4.0.
